packages/cli/src/utils/writeFile.ts

```diff
@@ -1,10 +1,11 @@
-import * as util from 'util';
-import * as _rimraf from 'rimraf';
-import * as _mkdirp from 'mkdirp';
-import * as path from 'path';
 import { TEMP_FOLDER } from '../constants';
 import { getFS } from '../dependencies';
 
+import path = require('path');
```

Comment:
why require over import?

Reply:
it's import but together with `require`. It's a weird typescript setting that I can't do it any other way. Tried multiple things. Didn't seem to work 🤷‍♀️

Reply:
oh well, maybe add a comment for that

Reply:
It's used in 1000 places :D

Reply:
oh... 😅  is there a rule of when we have to use it like that?

Reply:
yes, old module.exports

Reply:
Used to work well before with `import path from 'path'`, don't know what typescript does there.

Reply:
🤷